### PR TITLE
Prepare for new wiki build system

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

This is a first draft of the new wiki build system. It should make things a lot easier for devs. We will also provide a v2 of the wiki-cli to make things even easier ;)
I will keep this a draft until it is ready, but now if you want you can already look at what we are creating.
The wiki can be used by just running `yarn`, `yarn start` and navigating to http://localhost:3000/bee/welcome

TO-DOs:
- [x] Test
- [x] Add v2 of Wiki CLI

## Type of change

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] Enhancement (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Fix

### CODE REVIEW

**Comment by Dr-Electron** [comment]
This PR is now ready although there is still one little bug.
As of now `yarn iota-wiki start` doesn't work. We will fix it soon.
Until then you can use `yarn start` to test your docs

**Comment by Dr-Electron** [comment]
Please also approve that I based this on the right branch and this is the branch with the current (stable) docs

**Comment by Dr-Electron** [comment]
Ok, most of the things should work now :trollface:
You can test the docs locally with `yarn start`
I also added a test build to check for broken links and fixed currently broken links.

**Comment by Dr-Electron** [comment]
Also please inform the Wiki team (on discord) once you plan to merge as we need to update the wiki that day too :)

**Comment by Dr-Electron** [comment]
Added the cli build option to remove some workarounds and fixed the clip thingy

File: documentation/docs/getting_started/getting_started.md [hunk 1]
```diff
@@ -39,6 +39,6 @@ Bee exposes different functionalities on different ports:
 - 14265 TCP - REST HTTP API port (optional).
 - 8081 TCP - Dashboard (optional).
 
-The mentioned ports are important for flawless node operation. The REST HTTP API port is optional and is only needed if you want to offer access to your node's API. All ports can be customized inside the [config.toml](configuration.md) file.
+The mentioned ports are important for flawless node operation. The REST HTTP API port is optional and is only needed if you want to offer access to your node's API. All ports can be customized inside the [config.toml](configuration.mdx) file.
```
Comment (Alex6323) [inline documentation/docs/getting_started/getting_started.md:42]: should we mention `config.json`?

File: documentation/docs/setup_a_node.mdx [hunk 2]
```diff
@@ -187,7 +187,7 @@ cargo build --release
 
 ### Running
 
-Once you have downloaded and compiled Bee, you should copy make a copy of the example config file. Be sure to review and update your configuration. You can find more information on configuring bee in the [configuration section](configuration.md).
+Once you have downloaded and compiled Bee, you should copy make a copy of the example config file. Be sure to review and update your configuration. You can find more information on configuring bee in the [configuration section](configuration.mdx).
```
Comment (Alex6323) [inline documentation/docs/setup_a_node.mdx]:
```suggestion
Once you have downloaded and compiled Bee, you should make a copy of the example config file. Be sure to review and update your configuration. You can find more information on configuring Bee in the [configuration section](configuration.mdx).
```

File: documentation/docs/getting_started/getting_started.md (hunk 1 above)
Comment (Dr-Electron) [reply documentation/docs/getting_started/getting_started.md:42]: Sure. Should i fix that in this PR too?

**Review by thibault-martinez — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**